lib/pbench/agent/tool_meister.py

```diff
@@ -189,26 +186,25 @@ def _check_no_processes(self):
             )
 
     def start(self):
-        """Creates the background `screen` process running the tool's "start"
-        operation.
+        """Creates the background process running the tool's "start" operation.
         """
         self._check_no_processes()
-        # screen -dm -L -S \"${screen_name}\" ${pbench_bin}/tool-scripts/${name} --${action} --dir=${tool_output_dir} ${tool_opts[@]}
         args = [
-            screen_path,
-            "-dmS",
-            self.screen_name,
             f"{self.pbench_bin}/tool-scripts/{self.name}",
             "--start",
             f"--dir={self.tool_dir}",
             self.tool_opts,
         ]
         self.logger.info("%s: start_tool -- %s", self.name, " ".join(args))
-        self.start_process = subprocess.Popen(args)
+        o_file = self.tool_dir / f"tm-{self.name}-start.out"
+        e_file = self.tool_dir / f"tm-{self.name}-start.err"
+        with o_file.open("w") as ofp, e_file.open("w") as efp:
+            self.start_process = subprocess.Popen(
+                args, stdin=subprocess.DEVNULL, stdout=ofp, stderr=efp
+            )
```

Comment:
So this works? E.g., just attaching stdin to /dev/null? Or is this just a step to expose the fools who really "needed" screen in the first place?

Reply:
Yes, this works, and yes, hopefully it will help us show up problems with tools sooner than later.